nncf/experimental/openvino_native/graph/transformations/commands.py

```diff
@@ -84,3 +88,38 @@ def __init__(self, target_point: OVTargetPoint, quantizer_parameters: OVQuantize
     def union(self, other: 'TransformationCommand') -> 'TransformationCommand':
         # Have a look at nncf/torch/graph/transformations/commands/PTInsertionCommand
         raise NotImplementedError()
+
+
+class OVBiasCorrectionCommand(TransformationCommand):
```

Comment:
Could you please analyze how difficult it is to create common `BiasCorrectionCommand` and `ModelExtractionCommand` classes for ONNX and OV backends?

Reply:
It's not so difficult, I think. But let's do it in the next PR with the BiasCorrection algo update.